src/writer.ts

```diff
@@ -11,7 +12,15 @@ function intSequenceToString(intSequence) {
     .join('');
 }
 
-export async function handleSpaceCreated({ block, tx, event, mysql }) {
+export const handleSpaceCreated: CheckpointWriter = async ({
+  block,
+  tx,
+  event,
+  mysql,
+  instance
+}) => {
+  if (!event) return;
```

Comment:
A bit ugly, but event is optional so we have to check it. Will look into having two different paths so we can use different types there.